packages/core-typings/src/ILivechatVisitor.ts

```diff
@@ -20,6 +20,10 @@ export interface IVisitorEmail {
 	address: string;
 }
 
+interface ILivechatData {
+	[k: string]: any;
```

Comment:
could you check if unknown works?
```suggestion
	[k: string]: unknown;
```

Reply:
Typecheck didn't complain, so should work 👀